lawnchair/src/app/lawnchair/qsb/providers/GitHub.kt

```diff
@@ -0,0 +1,12 @@
+package app.lawnchair.qsb.providers
+
+import com.android.launcher3.R
+
+object GitHub : QsbSearchProvider(
+    id = "github",
+    name = R.string.search_provider_github,
+    icon = R.drawable.ic_github,
+    // todo: Add packageName & className back after https://github.com/orgs/community/discussions/39678 is resolved.
+    packageName = "",
+    website = "https://github.com/search"
+)
```

Comment:
The type should be `QsbSearchProviderType.WEBSITE` but you haven't changed the default value for `type` which is `QsbSearchProviderType.APP_AND_WEBSITE`.

Reply:
Yep, I've noticed that, I was just wondering if I should keep `APP_AND_WEBSITE` or `WEBSITE`?

Reply:
eef85bfd74215b4046e1c4ceffc2992ca73af1af